MonoGame.Framework/Quaternion.cs

```diff
@@ -776,7 +1146,11 @@ public static void Normalize(ref Quaternion quaternion, out Quaternion result)
 
         }
 
-
+        /// <summary>
+        /// Inverts values in the specified <see cref="Quaternion"/>.
+        /// </summary>
+        /// <param name="quaternion">Source <see cref="Quaternion"/> on the right of the sub sign.</param>
+        /// <returns>Result of the inversion.</returns>
         public static Quaternion operator -(Quaternion quaternion)
```

Comment:
This is negation... not inversion... those are two very different things.  Use the docs for `Negate` here.